main/pid.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
+)
+
+// GetProcessStartTime returns the start time of the active process if still active
+func GetProcessStartTime(pid int) (string, error) {
```

Comment:
Is the process start time guaranteed to be in a parseable format?

Reply:
Oh, I see that its just being used for equality checks

Reply:
The format we use comes from: ps -o lstart= -p [pid]
It was never parsed back to date object. We only generate the string the same way and use it to compare.